### PR TITLE
output: Messaging now represent the proper product.

### PR DESCRIPTION
**Pull request by ashcrow**
- Renamed openshift* binaries to atomic-enterprise*
- Error messages that are not OpenShift specific now reference Atomic Enterprise
- CLI flags descriptions updated

Verification:
- unittests all passed
- cmd tests do not pass, but failed at the same point before this PR (will open an issue)
- RPM packages all build and lint as they did before

### CODE REVIEW

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -586,7 +586,7 @@ oc delete bc/ruby-sample-build-validtag
 oc delete bc/ruby-sample-build-invalidtag
 
 # Test admin manage-node operations
-[ "$(openshift admin manage-node --help 2>&1 | grep 'Manage node operations')" ]
+[ "$(atomic-enterprise admin manage-node --help 2>&1 | grep 'Manage node operations')" ]
```
Comment (aveshagarwal) [inline hack/test-cmd.sh]: Might be changed to oadm.

File: cmd/genconversion/conversion.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func main() {
 	// TODO(wojtek-t): Change the overwrites to a flag.
 	generator.OverwritePackage(*version, "")
 	for _, knownType := range api.Scheme.KnownTypes(*version) {
-		if !strings.Contains(knownType.PkgPath(), "openshift/origin") {
+		if !strings.Contains(knownType.PkgPath(), "atomic-enterprise/origin") {
```
Comment (aveshagarwal) [inline cmd/genconversion/conversion.go]: In the deep_copy.go "openshift/origin" is changed to "projectatomic/atomic-enterprise" which seems more correct. Might be good to double check it.

File: hack/common.sh [hunk 3]
```diff
@@ -53,11 +53,9 @@ readonly OS_ALL_BINARIES=("${OS_ALL_TARGETS[@]##*/}")
 
 #If you update this list, be sure to get the images/origin/Dockerfile
 readonly OPENSHIFT_BINARY_SYMLINKS=(
-  openshift-router
-  openshift-deploy
-  openshift-sti-build
-  openshift-docker-build
-  openshift-gitserver
+  atomic-enterpriset-router
```
Comment (aveshagarwal) [inline hack/common.sh]: typo atomic-enterpriset-router

File: hack/test-cmd.sh (hunk 1 above)
Comment (ashcrow) [inline hack/test-cmd.sh]: I took a look and both openshift admin/atomic-enterprise admin and oadm produce the same results.

Comment (aveshagarwal) [inline hack/test-cmd.sh]: yes that are same, but for consistency, I recommended oadm as thats what is being used everywhere.

File: pkg/auth/oauth/handlers/default_auth_handler.go [hunk 4]
```diff
@@ -38,7 +38,7 @@ const (
 	// WarningHeaderMiscCode is the code for "Miscellaneous warning", which may be displayed to human users
 	WarningHeaderMiscCode = "199"
 	// WarningHeaderOpenShiftSource is the name of the agent adding the warning header
-	WarningHeaderOpenShiftSource = "OpenShift"
+	WarningHeaderOpenShiftSource = "AtomicEnterprise"
```
Comment (aveshagarwal) [inline pkg/auth/oauth/handlers/default_auth_handler.go]: I would prefer Atomic-Enterprise, but your choice.

File: pkg/build/admission/admission.go [hunk 5]
```diff
@@ -20,7 +20,7 @@ func init() {
 	admission.RegisterPlugin("BuildByStrategy", func(c kclient.Interface, config io.Reader) (admission.Interface, error) {
 		osClient, ok := c.(client.Interface)
 		if !ok {
-			return nil, errors.New("client is not an OpenShift client")
+			return nil, errors.New("client is not an Atomic Enterpries client")
```
Comment (aveshagarwal) [inline pkg/build/admission/admission.go]: "Enterpries" typo or intentional?

File: pkg/cmd/cli/cli.go [hunk 6]
```diff
@@ -29,7 +29,7 @@ To create a new application, you can use the example app source. Login to your s
 run new-app:
 
   $ %[1]s login
-  $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
+  $ %[1]s new-app atomicenterprise/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
```
Comment (aveshagarwal) [inline pkg/cmd/cli/cli.go]: In hack/test-cmd.sh, its atomic-enterprise/ruby-20-centos7, might double check.

File: pkg/cmd/cli/cmd/newapp.go [hunk 7]
```diff
@@ -48,7 +48,7 @@ application is created.`
   $ %[1]s new-app . --docker-image=repo/langimage
 
   // Create a Ruby application based on the provided [image]~[source code] combination
-  $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
+  $ %[1]s new-app atomicenterprise/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
```
Comment (aveshagarwal) [inline pkg/cmd/cli/cmd/newapp.go]: In hack/test-cmd.sh, its atomic-enterprise/ruby-20-centos7, might double check.

**Comment by ashcrow** [comment]
Updated based on @aveshagarwal's notes.

**Comment by themurph** [comment]
@aveshagarwal In hack/test-cmd.sh, its atomicenterprise/ruby-20-centos7, not atomic-enterprise/ruby-20-centos7. All other changes look correct, merging now and will watch builds to confirm.

File: hack/test-cmd.sh [hunk 8]
```diff
@@ -241,17 +241,17 @@ oc get templates
...
-[ "$(openshift ex tokens)" ]
-[ "$(openshift admin policy  2>&1)" ]
-[ "$(openshift kubectl 2>&1)" ]
-[ "$(openshift kube 2>&1)" ]
-[ "$(openshift admin 2>&1)" ]
-[ "$(openshift start kubernetes 2>&1)" ]
+[ "$(atomic-enterprise 2>&1)" ]
+[ "$(atomic-enterprise cli)" ]
+[ "$(atomic-enterprise ex)" ]
+[ "$(oadm config 2>&1)" ]
```
Comment (miminar) [inline hack/test-cmd.sh]: I believe `atomic-enterprise admin config` should work as well.

File: hack/test-cmd.sh [hunk 9]
```diff
@@ -395,7 +395,7 @@ oc create -f examples/image-streams/image-streams-centos7.json
 [ -n "$(oc get imageStreams mongodb -t "{{.status.dockerImageRepository}}")" ]
 # verify the image repository had its tags populated
 [ -n "$(oc get imageStreams wildfly -t "{{.status.tags.latest}}")" ]
-[ -n "$(oc get imageStreams wildfly -t "{{ index .metadata.annotations \"openshift.io/image.dockerRepositoryCheck\"}}")" ]
+[ -n "$(oc get imageStreams wildfly -t "{{ index .metadata.annotations \"atomic-enterprise.io/image.dockerRepositoryCheck\"}}")" ]
```
Comment (miminar) [inline hack/test-cmd.sh]: This domain is not yet registered. I'm not sure of its impact though.

File: hack/test-cmd.sh [hunk 10]
```diff
@@ -576,7 +576,7 @@ oc create -f test/integration/fixtures/test-buildcli.json
 # a build for which there is not an upstream tag in the corresponding imagerepo, so
 # the build should use the image field as defined in the buildconfig
 started=$(oc start-build ruby-sample-build-invalidtag)
-oc describe build ${started} | grep openshift/ruby-20-centos7$
+oc describe build ${started} | grep atomicenterprise/ruby-20-centos7$
```
Comment (miminar) [inline hack/test-cmd.sh]: `atomicenterprise/ruby-20-centos7` isn't published on docker hub.

Comment (ashcrow) [inline hack/test-cmd.sh]: This was to be part of the follow on work but it's been split. Will have a follow on PR.

**Pull request merged**